### PR TITLE
Add Kusari

### PR DESCRIPTION
**Pull request by masterdubs**
Add kusari network to live network.

### CODE REVIEW

File: packages/apps-config/src/api/spec/index.ts [hunk 1]
```diff
@@ -190,6 +191,7 @@ const spec: Record<string, OverrideBundleDefinition> = {
   konomi,
   kpron,
   kulupu,
+  kusari: kusari,
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/index.ts:194]:
```suggestion
  kusari,
```

File: packages/apps/package.json [hunk 2]
```diff
@@ -11,20 +11,20 @@
...
+    "@polkadot/apps-config": "^0.98.2-136",
+    "@polkadot/apps-routing": "^0.98.2-136",
     "@polkadot/dev": "^0.63.37",
-    "@polkadot/react-api": "^0.98.2-133",
-    "@polkadot/react-components": "^0.98.2-133",
-    "@polkadot/react-hooks": "^0.98.2-133",
-    "@polkadot/react-params": "^0.98.2-133",
-    "@polkadot/react-query": "^0.98.2-133",
-    "@polkadot/react-signer": "^0.98.2-133",
+    "@polkadot/react-api": "^0.98.2-136",
```
Comment (jacogr) [inline packages/apps/package.json:22]: Not sure where these come from. Should not be there.

**Review by jacogr — COMMENTED** [review]
2 small comments - the only thing are all these version updates, not sure why they are in the PR, it gets applied on CI.

File: packages/apps/package.json (hunk 2 above)
Comment (masterdubs) [reply packages/apps/package.json:22]: Yes pretty strange, I tried to build it locally to be sure all is ok, is it it auto-incremented at this time maybe ?

Comment (masterdubs) [reply packages/apps/package.json:22]: Should I redo a commit removing all the version changes ?

Comment (jacogr) [reply packages/apps/package.json:22]: Yes, please do :) Sorry about that, generally they are only incremented with the gh release build. This one throws it a bit out :)

Comment (masterdubs) [reply packages/apps/package.json:22]: closing, pushed a new one :)

**Pull request closed without merging**